unitTests/magicbox/SuggestionsManagerTest.ts

```diff
@@ -65,6 +65,12 @@ export function SuggestionsManagerTest() {
       expect(selectedWithKeyboard).toBe(suggestion.el);
     });
 
+    it('clearKeyboardFocusedElement sets the keyboard focused element to null', () => {
+      suggestionManager.moveDown();
+      suggestionManager.clearKeyboardFocusedElement();
```

Comment:
Hmm, the test looks good but I would feel even more confident if we could assert that the focused element is not `null` before calling `clearKeyboardFocusedElement`. What do you think?

Reply:
The previous test does that, the way to get the element is to call selectAndReturnKeyboardFocusedElement which sets it to null after return, so calling it twice would defeat the purpose.